R/board_rsconnect.R

```diff
@@ -601,7 +617,12 @@ rsc_GET <- function(board, path, query = NULL, ...) {
     ...
   )
   rsc_check_status(req)
-  httr::content(req)
+  content <- httr::content(req)
+  if (length(content) && is.list(content) && is.list(content[[1]])) {
```

Comment:
I think this is the right place, since the problem actually occurs in the line 620 in `httr::content(req)`.

Fixing it here fixes the problem when it occurs.  If we push the fix downstream, you will have to potentially fix in multiple places.

Reply:
The currently code assumes that every response from the server has `content_url` in the first element of the response, which seems odd to me. 

If look at `board_rsconnect.R`, the only place I can see `content_url` being read is `rsc_content_find()` (which saves it in the cache), so can't we fix it there?

Reply:
This pointer to `rsc_content_find()` is helpful.  I'll investigate.